service/service.go

```diff
@@ -326,3 +310,61 @@ func (svc *Service) Shutdown(topic string) {
 	}
 	log.Info("Consumer successfully closed", log.Data{"topic": svc.Topic})
 }
+
+// Creates Eshu resources
+func (svc *Service) getEshuResources(
+	message *sarama.ConsumerMessage,
+	paymentResponse data.PaymentResponse,
+	paymentDetailsResponse data.PaymentDetailsResponse,
+	paymentId string) []models.EshuResourceDao {
+
+	eshus, err := svc.Transformer.GetEshuResources(paymentResponse, paymentDetailsResponse, paymentId)
+	if err != nil {
+		log.Error(err, log.Data{"message_offset": message.Offset})
+		_ = svc.HandleError(err, message.Offset, &paymentDetailsResponse)
```

Comment:
`_ =` isn't needed here.

Reply:
I actually introduced that because otherwise the `Goland Code Analysis` issues this warning:

`Warning:(324, 7) Unhandled error`

I think by using the `_` we are saying we know there is an unused return value variable here?

Reply:
If you want to change an error to be ignored explicitly by adding `_=` you should do this for all calls to `svc.HandleError` in this file for consistency.